docs/user/docs/sessions/jupyter/collaboration.md

```diff
@@ -0,0 +1,62 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Collaboration with Jupyter Notebooks
+
+Collaborating on Jupyter notebooks is a common requirement in various
+workflows. This guide describes two main approaches to collaboration:
+individual sessions and project-level collaboration.
+
+## Collaboration in Individual Sessions
+
+If you have a notebook in your personal workspace and another user wants to
+connect to your sessions, follow these steps:
+
+### Sharing a Notebook Session
+
+1. **Request a Jupyter Session**: As usual, request a Jupyter session to start.
+2. **Connect to the Session**: Open the requested session.
+3. **Share the Session**: Click on the "Share" button on the top right of the
+   Jupyter session interface.
+4. **Include the Token**: Tick the option "Include the token in the URL".
+5. **Warning! Security Risk**: Be aware that anyone with access to the URL can
+   access your Jupyter session until termination. To revoke access, terminate
+   your session in the Capella Collaboration Manager and optionally request a
+   new session.
+6. **Share the Link**: Distribute the link to all users who should have access.
+   They will be able to open your notebooks and concurrently edit them.
+
+#### Instructional Video
+
+Here's a video that visually guides you through these steps:
+
+<video controls>
+  <source src="../jupyter-collaboration.mp4" type="video/mp4">
+</video>
+
+## Collaboration on Project Level
+
+If you need a shared workspace for notebooks at the project level, you can
+create a shared space accessible by all project members.
```

Comment:
IMO this (user) documentation should only land once the feature actually becomes visible to users. The most important thing here being that it won't go into production's docs before the actual feature can go into production as well. If we can make sure that's the case, it should be fine keeping it here, even though the actual (frontend and backend) functionality isn't quite there yet. Although personally I would rather like to see these docs together with the frontend update :)

Reply:
Good point, I'll move this part to a new PR, where I'll add the real feature later on.

Reply:
The documentation commit is edited and does only include the collaboration part. 

The file-share part is moved to a new PR, where I'll add the file-share mounting soon: https://github.com/DSD-DBS/capella-collab-manager/pull/918